### PR TITLE
Use the Configured Defaults when Outside the EU (into develop branch)

### PR DESCRIPTION
**Pull request by dhurlburtusa**
Oops! Sorry about the mix up on the branches.

### CODE REVIEW

**Comment by Bilal-S** [comment]
@dhurlburtusa 
I will wait until all the fixes are in this branch before merge with dev.
Then, I will revert the previous PR from main

**Review by Bilal-S — CHANGES_REQUESTED** [review]
This breaks the display altogether:

```
xcoobee-cookie-kit.min.js:27 TypeError: w.filter is not a function
    at a.value (xcoobee-cookie-kit.min.js:35)
    at Fa (xcoobee-cookie-kit.min.js:27)
    at Ia (xcoobee-cookie-kit.min.js:27)
    at ml (xcoobee-cookie-kit.min.js:27)
    at cu (xcoobee-cookie-kit.min.js:27)
    at lu (xcoobee-cookie-kit.min.js:27)
    at Zl (xcoobee-cookie-kit.min.js:27)
    at xcoobee-cookie-kit.min.js:27
    at t.unstable_runWithPriority (xcoobee-cookie-kit.min.js:35)
    at Ho (xcoobee-cookie-kit.min.js:27)

```

Can we review together next week?

**Comment by dhurlburtusa** [comment]
[Cookie Kit: ignores checked by default](https://trello.com/c/FBN4WVSG/4352-cookie-kit-ignores-checked-by-default)

**Pull request merged**
